computer-vision/README.md

```diff
@@ -0,0 +1,42 @@
+# Computer Vision
+
+The repository shall focus on the traditional aspects of computer vision, particularly multiview geometry. 
+
+## Pre - Requisites
+1. It is expected that one is familiar with some of the concepts of Linear Algebra. No course can beat the one provided by [Gilbert Strang](https://www.youtube.com/playlist?list=PLE7DDD91010BC51F8). Make sure that one sticks patiently as some of the concepts might seem daunting initially. For gaining some motivation and intuition for some of the concepts, one can refer to the [Essence of Linear Algebra](https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab).
+2. For implementation of some of the concepts, [OpenCV](https://opencv.org/releases/) will help you. Note that you are required to have some background in python. There are many tutorials available, such as [the one by Corey Schafer](https://www.youtube.com/watch?v=YYXdXT2l-Gg&list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU), or [by sentdex](https://www.youtube.com/playlist?list=PLQVvvaa0QuDe8XSftW-RAxdo6OmaeL85M). 
+
+## Basic Roadmap
+
+1. To learn basic concepts that are used in image processing and computer vision, [Introduction to Computer Vision](https://www.udacity.com/course/introduction-to-computer-vision--ud810) is a good place to start. The course starts with introducing various image processing techniques required, before moving on to introducing camera geometry, such as the pinhole model, intrinsic and extrinsic camera parameters and analysing the point correspondences between two views of the same scene. The course moves on to introduce the concept of features, scale invariance and then introducing the state of the art method, Scale Invariant Feature Transform (SIFT). One can complete the whole course in order to get an idea on different computer vision and image processing techniques. 
+2. For discussing the feature detection and matching techniques, one can have a look at the [documentation](https://docs.opencv.org/3.4.2/db/d27/tutorial_py_table_of_contents_feature2d.html) provided by OpenCV. The documentation starts with the introduction to the concept of features, followed by discussing various techniques to obtain the feature points. 
+
+## Other Lecture Series
+
+To look for more advanced concepts, one can refer to the following lecture courses selectively.
+
+1. [Computer Vision for Visual Effects, by Rich Radke, Rensselaer Polytechnique Institute](https://www.youtube.com/playlist?list=PLuh62Q4Sv7BUJlKlt84HFqSWfW36MDd5a)
+2. [Photogrammetry Course by Cyrill Stachniss, University of Bonn](https://www.youtube.com/playlist?list=PLgnQpQtFTOGRsi5vzy9PiQpNWHjq-bKN1)
+3. [Multiple View Geometry by Daniel Cremers, Technical University Munich](https://www.youtube.com/playlist?list=PLTBdjV_4f-EJn6udZ34tht9EVIW7lbeo4)
+4. [Computer Vision by Mubarak Shah, University of Central Florida](https://www.youtube.com/playlist?list=PLd3hlSJsX_Imk_BPmB_H3AQjFKZS9XgZm)
```

Comment:
This course is also an Intro to CV. Make Intro to CV course section and put udacity and Mubarak shah course in that